lib/facebook.ex

```diff
@@ -309,6 +330,18 @@ defmodule Facebook do
   end
 
   """
+  Extract the access token from the access token response
+  """
+  defp getAccessToken(access_token_response) do
+    case access_token_response do
+      {:json, %{"error" => error}} -> %{"error" => error}
+      {:json, info_map} ->
+        info_map
+          |> Map.fetch!("access_token")
```

Comment:
Can you return the whole map here? I guess the `expires_in` param is interesting too.